### PR TITLE
Czech translation attempt 1

### PR DESCRIPTION
**Pull request by vojtaruzek**
Hopefully, it will actually add translation to Replimat

### CODE REVIEW

File: Common/Languages/Czech/Keyed/Replimat.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LanguageData>
+
```
Comment (sumghai) [inline Common/Languages/Czech/Keyed/Replimat.xml:3]: Looks like you forgot to translate this file ;)

**Review by sumghai — CHANGES_REQUESTED** [review]
@vojtaruzek - Most of the translations look good! 

You probably forgot to translate `Common/Languages/Czech/Keyed/Replimat.xml`, though.

To fix this, simply push a new commit with the fixed `Keyed/Replimat.xml` file to your repository, and GitHub will automatically add it to this pull request.

**Comment by vojtaruzek** [comment]
It should be done now.

**Review by sumghai — APPROVED** [review]
Cool, thanks for your submission! I'll merge this ASAP, and release an update over the next few days.

**Pull request merged**